tools/defaults.yaml

```diff
@@ -0,0 +1,4 @@
+"build":
```

Comment:
We can do this, but I'd rather have this in the CI file itself to keep things relatively contained. Do you think we'll want to mess with this in enough volume where having the separate file would be beneficial?

Reply:
It might be useful to consulate all the CI specific colcon config, without having to bake it into the script functions. I could move this yaml file into the .circleci folder so they are closer together?

Reply:
That would also work

Reply:
Done.